public/apps/configuration/utils/panel-with-header.tsx

```diff
@@ -31,14 +33,17 @@ export function PanelWithHeader(props: PanelWithHeaderDeps) {
         <h3>
           {props.headerText}
           {props.optional && <i> - optional</i>}
+          {props.count?.toString() && (
+            <span style={{ color: '#687078', fontWeight: 'normal' }}> ({props.count})</span>
+          )}
         </h3>
       </EuiTitle>
-      <EuiText size="xs" color="subdued">
+      <EuiText size="xs" color="subdued" style={{ maxWidth: Widths.width75 }}>
```

Comment:
I would suggest have a `_index.scss` file to define the style and use cssClass to use the style

Reply:
Moved the style code to `_index.scss` and used `className` to use style.